server/routes/users.js

```diff
@@ -0,0 +1,66 @@
+const express = require('express')
+const mongoose = require('mongoose')
+const consola = require('consola')
+const { addUser } = require('../seeder/index')
+
+const Users = mongoose.model('User')
+const router = express.Router()
+
+const sanitiseUser = userObj => {
+  return {
```

Comment:
The user object may change in the future, it would be better to just `delete` password fields and return the rest

Reply:
I tried that :'( Will try again but it wasn't working with the method i tried